src/Dashboard.vue

```diff
@@ -11,11 +11,11 @@
         <v-spacer></v-spacer>
         <div class="d-flex">
           <FundsCard
-            v-if="getCredentials() && getCredentials().twinID != 0"
+            v-if="$store.state.credentials.initialized"
```

Comment:
I think you can store $store.state.credentials in a variable called credentials, for instance, then use it here like credentials.initialized. This could be done in all the files.

Reply:
I think it's better the way it is as it's obvious that we get data from the store, I want to make it clear to who will work on it in the future how to use it